### ISSUE TITLE
apache-tomcat-8.0.47 cannot "jpda start" on openjdk8-1.8.152nb1

### ISSUE THREAD

**Issue by idodeclare** [TritonDataCenter/pkgsrc#86]
Joyent pkgsrc ships Tomcat 8, but`/opt/local/share/tomcat/bin/catalina.sh` cannot be started successfully with `jpda start` (Start Catalina under JPDA debugger) due to the following:
```
ld.so.1: java: fatal: relocation error: file /opt/local/java/openjdk8/jre/lib/amd64/libnpt.so: symbol libiconv_open: referenced symbol not found
```

**Comment by mamash** [comment]
Yep, looks like a bug in the OpenJDK8 build process, I'll get it fixed.

**Comment by idodeclare** [comment]
Thank you kindly.

**Comment by mamash** [comment]
This should be fixed now in openjdk8-1.8.162nb1 in 2017Q4, I was able to 'lpda start' successfully. Can you update your openjdk8 package and verify?

**Comment by idodeclare** [comment]
Thank you, @mamash — I can confirm it starts up now.
